.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ on: # yamllint disable-line rule:truthy rule:comments
   pull_request: ~
 
 env:
-  APP_NAME: "nautobot-app-golden-config"
```

Comment:
this seems wrong to me.  Just want second set of eyes.

Reply:
This needs to match the app name in the docker image [here](https://github.com/nautobot/nautobot-app-golden-config/blob/00d018df4c23cc689b234afe32d0edca8359fb07/development/docker-compose.base.yml#L10)